app/src/main/java/com/github/studydistractor/sdp/login/LoginActivity.kt

```diff
@@ -113,7 +113,6 @@ class LoginActivity : Hilt_LoginActivity() {
                             } else {
                                 Toast.makeText(this@LoginActivity, "Login Failed", Toast.LENGTH_SHORT).show()
                                 Log.d(TAG, "signInWithEmail:failed")
-                                Log.d(TAG, "email: " + emailstr + ", password: " + passwordstr)
```

Comment:
Actually this is useful, you should keep it

Reply:
Not sure to let the password in the console is a good idea but I'll keep it then

Reply:
Actually it only goes to the console in debug mode